module/Paprika/src/Model/Product/RAM.php

```diff
@@ -0,0 +1,180 @@
+<?php
+declare(strict_types=1);
+namespace Paprika\Model\Product;
+use DomainException;
+use Zend\Filter\StringTrim;
+use Zend\Filter\StripTags;
+use Zend\Filter\ToInt;
+use Zend\InputFilter\InputFilter;
+use Zend\InputFilter\InputFilterAwareInterface;
+use Zend\InputFilter\InputFilterInterface;
+use Zend\Validator\StringLength;
+/**
+ * Class Product

```

Comment:
Richtige Klassenname und Autor 
```suggestion
 * Class Ram
 *
 * @author Jonas <jonas@example>
```